### PR TITLE
feature/detect-persons-changes

### PR DESCRIPTION
**Pull request by dphoria**
### Link to Relevant Issue

This pull request is part of #135 

#### `extract_persons`, `compare_persons`

Helpers to find out when `ScraperStaticData` potentially needs updating. No scraper is using these helper methods for the time being.

Not a complete solution by any means, but should be helpful in detecting obvious council changes.

#### Portland, OR

Added [basic info for] Rene Gonzalez and updated Dan Ryan.

### CODE REVIEW

**Comment by dphoria** [comment]
Unfortunately not a small PR but a lot of it is tests verifying that persons extracting and comparing work as desired.

**Comment by dphoria** [comment]
In a later version might actually not call these helpers in `get_events`. No need to check all the time.

File: cdp_scrapers/tests/test_scraper_utils.py [hunk 1]
```diff
@@ -21,3 +39,332 @@
...
+        s = PortlandScraper()
+        events = s.get_events(begin=begin, end=end)
+        scraped_persons = extract_persons(events)
+        old_new = compare_persons(
+            scraped_persons,
+            PORTLAND_STATIC_DATA.persons.values(),
+            PORTLAND_STATIC_DATA.primary_bodies.values(),
+        )
+
+        assert person in old_new.old_names
```
Comment (dphoria) [inline cdp_scrapers/tests/test_scraper_utils.py:370]: These 2 tests provide example use cases.

**Comment by dphoria** [comment]
The test failure on macOS is due to the server hanging up on us. I think we can accept them.

```
http.client.RemoteDisconnected: Remote end closed connection without response
```

File: cdp_scrapers/tests/test_scraper_utils.py (hunk 1 above)
Comment (evamaxfield) [reply cdp_scrapers/tests/test_scraper_utils.py:370]: Gorgeous

**Review by evamaxfield — APPROVED** [review]
Beautiful throughout! Merge whenever!!

**Pull request merged**